src/components/FinishAccount/FinishAccount.jsx

```diff
@@ -25,42 +35,166 @@ const FinishAccount = ({ inviteId, data }) => {
       setErrorMessage(err.message);
     }
   };
+  */
+
+  const passwordRulesTooltipText =
+    '\u2022 Must be at least 8 characters long' +
+    '<br />' +
+    '\u2022 Include at least one uppercase and lowercase letter' +
+    '<br />' +
+    '\u2022 Include at least 1 number' +
+    '<br />' +
+    '\u2022 Include one of these set of special characters [@$!%*?&]';
+
+  const [newPasswordShown, setNewPasswordShown] = useState(false);
+  const toggleNewPasswordVisibility = () => {
+    setNewPasswordShown(!newPasswordShown);
+  };
+
+  const [reenterPasswordShown, setReenterPasswordShown] = useState(false);
+  const toggleReenterPasswordVisibility = () => {
+    setReenterPasswordShown(!reenterPasswordShown);
+  };
+
+  const passwordRegExp = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,255}$/;
+  const passValidationSchema = Yup.object().shape({
+    currentPassword: Yup.string().required('Current password is required'),
+    newPassword: Yup.string()
+      .required('Please enter your new password')
+      .matches(passwordRegExp, 'Password is invalid'),
+    reenterNewPassword: Yup.string()
+      .required('Please re-enter your new password')
+      .oneOf([Yup.ref('newPassword'), null], 'Passwords do not match!'),
+  });
+
+  const {
+    register: registerPass,
+    handleSubmit,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'handleSubmit' is assigned a value but never used.